schemas/chemotherapy.json

```diff
@@ -50,19 +50,32 @@
       }
     },
     {
-      "name": "chemotherapy_drug_name",
-      "description": "Name of agent or drug administered to patient as part of the chemotherapy treatment regimen.",
+      "name": "drug_rxnormid",
+      "description": "RxNormID assigned to the treatment regimen.",
       "valueType": "string",
-      "restrictions": {
-        "required": true,
-        "codeList": ["Placeholder list 1", "Need list", "Still need it"]
+      "meta": {
+        "validationDependency": true,
+        "core": true,
+        "notes": "This field uses a controlled vocabulary gathered from the DrugBank database of drug names. Either this or a valid drug_name must be provided.",
+        "displayName": "RXNormID"
       },
+      "restrictions":{
+        "required": true
+      }
+    },
+    {
+      "name": "drug_name",
+      "description": "Name of agent or drug administered to patient as part of the treatment regimen.",
+      "valueType": "string",
       "meta": {
         "validationDependency": true,
         "core": true,
-        "notes": "This field uses a controlled vocabulary gathered from the DrugBank database of drug names.",
-        "examples": "Allopurinol, Dronabinol, Sucralfate, Lapatinib",
+        "notes": "This field uses a controlled vocabulary gathered from the DrugBank database of drug names. Either this or a valid drug_rxnormid must be provided.",
         "displayName": "Chemotherapy Drug Name"
+      },
+      ,
```

Comment:
double comma

Reply:
sorry working with a new editor and still trying to notice when it highlights errors! fixed